client/src/images/checked.svg

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
Optionally I would recommend clean and optimize this `svgs` using [ImageOptim](https://imageoptim.com/mac):
```bash 
brew cask install imageoptim
```

Just drag and drop the svg on the desktop tool and will optimize and replace instantly the file/s you dropped.